test/dummy-aave-trigger.ts

```diff
@@ -0,0 +1,175 @@
+        const tx = await (await DPMFactory['createAccount(address)'](executorAddress)).wait()
+        const [AccountCreatedEvent] = getEvents(tx, DPMFactory.interface.getEvent('AccountCreated'))
+        console.log('account created')
+        DPMAccount = await hre.ethers.getContractAt('IAccountImplementation', AccountCreatedEvent.args.proxy)
+        const signer = await utils.impersonate('0x060c23F67FEBb04F4b5d5c205633a04005985a94')
+        console.log('Imperosnated signer', await signer.getAddress())
+        console.log('executorAddress', executorAddress)
+
+        AaveCommandInstance = (await utils.deployContract(hre.ethers.getContractFactory('DummyAaveWithdrawCommand'), [
+            system.aaveProxyActions!.address,
+            utils.addresses.USDC_AAVE,
+        ])) as DummyAaveWithdrawCommand
+
+        await system.serviceRegistry.addNamedService(
+            getCommandHash(TriggerType.SimpleAAVESell),
+            AaveCommandInstance!.address,
+        )
+        await system.serviceRegistry.addNamedService(
+            getAdapterNameHash(AaveCommandInstance.address),
+            system.dpmAdapter!.address,
+        )
+        await system.serviceRegistry.addNamedService(
+            getExecuteAdapterNameHash(AaveCommandInstance.address),
+            system.aaveAdapter!.address,
+        )
+
+        console.log('DPMAccount', await DPMAccount.address)
+
+        await DPMGuard.connect(signer).setWhitelist(system.aaveProxyActions!.address, true)
+        await DPMGuard.connect(signer).setWhitelist(system.automationBot.address, true)
+        console.log('APA whitelisted', await system.aaveProxyActions?.aave())
+        const encodedData = system.aaveProxyActions!.interface.encodeFunctionData('openPosition')
+        await (
+            await DPMAccount.connect(hre.ethers.provider.getSigner(0)).execute(
+                system.aaveProxyActions!.address!,
+                encodedData,
+                {
+                    gasLimit: 10000000,
+                    value: hre.ethers.BigNumber.from(10).mul(hre.ethers.BigNumber.from(10).pow(18)),
+                },
+            )
+        ).wait()
+
+        const args = [DPMAccount.address, TriggerType.SimpleAAVESell, '1000000', 1800, DPMAccount.address]
+        const types = getDefinitionForCommandType(CommandContractType.SimpleAAVESellCommand)
+        triggerData = EthUtils.defaultAbiCoder.encode(types, args)
+    })
+
+    beforeEach(async () => {
+        snapshotId = await hre.ethers.provider.send('evm_snapshot', [])
+    })
+
+    afterEach(async () => {
+        await hre.ethers.provider.send('evm_revert', [snapshotId])
+    })
+
+    it('should be able to add Trigger', async () => {
+        const dataToSupply = AutomationBotInstance.interface.encodeFunctionData('addTriggers', [
+            TriggerGroupType.SingleTrigger,
+            [true],
+            [0],
+            [triggerData],
+            [TriggerType.SimpleAAVESell],
```

Comment:
there's not `SimpleAAVESell` in `TriggerType`

Reply:
"@oasisdex/automation": "1.2.7-alpha.2"

this is package.json